apps/web/src/views/BuyCrypto/components/AccordianDropdown/AccordianItem.tsx

```diff
@@ -38,7 +39,7 @@ const FeeItem = ({
     currentLanguage: { locale },
   } = useTranslation()
 
-  if (provider === 'Mercuryo' && index === 1) return <></>
+  if (provider === 'Mercuryo' && (index === 1 || index === 2)) return <></>
```

Comment:
What does it mean with `index === 1` and `index === 2`. Is there a more meaningful way to describe the condition? Also if there's nothing to return we can simply return `null`